### PR TITLE
added menu and new pricing model page

### PR DESCRIPTION
**Pull request by Milstein**

### CODE REVIEW

**Review by joachimweyl — APPROVED** [review]
looks good, just know that the document you are pulling this information from is in a process of change so may require updates in the near future.

File: docs/get-started/cost-billing/billing-faqs.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+??? question "2. If I have an issue with my bill, who do I contact?"
+
+    Please send your requests by emailing us at
+    [help@nerc.mghpcc.org](mailto:help@nerc.mghpcc.org?subject=NERC%20MOU%20Question)
+    or, by submitting a new ticket at [the NERC's Support Ticketing System (osTicket)](https://mghpcc.supportsystem.com/open.php).
+
+??? question "3. How do I control costs?"
+
+    Upon creating a project, you will set these resource limits (quotas) for
```
Comment (joachimweyl) [inline docs/get-started/cost-billing/billing-faqs.md:33]: is it useful to add something about ColdFront here?

File: docs/get-started/cost-billing/billing-faqs.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    OpenStack (VMs),OpenShift (containers), and storage.  This is the maximum
+    amount of resources you can consume at one time.
+
+??? question "4. Are we only invoicing for CPUs/GPUs only when the VM or Pod is active?"
```
Comment (joachimweyl) [inline docs/get-started/cost-billing/billing-faqs.md:37]: remove one of the only's

File: docs/get-started/cost-billing/images/pricing-monthly-invoicing-data.png
[the remark this answers is not in the document]
Comment (Milstein) [reply docs/get-started/cost-billing/images/pricing-monthly-invoicing-data.png:1]: I removed these unused images for now.

**Pull request merged**